cmd/genji/shell/shell.go

```diff
@@ -260,6 +260,18 @@ func (sh *Shell) runCommand(in string) error {
 			return err
 		}
 		return runIndexesCmd(db, cmd)
+	case ".dump":
+		db, err := sh.getDB()
+		if err != nil {
+			return err
+		}
+
+		t := ""
+		if len(cmd) == 2 {
```

Comment:
What happens if I write `.dump foo bar baz`?

Reply:
What do you want? Should I print the content of the three args ? And so change it to `runDumpCmd(db, cmd[1:]) `

Reply:
Yeah that's a good idea

Reply:
Ok What about the error handling. Let's admit `.dump foo bar baz` where `bar` doesn't exists?

Reply:
Good question, I think we can ignore them and display BEGIN then COMMIT.

```sql
.dump tableThatDoesntExist
BEGIN TRANSACTION:
COMMIT;
```